diff -aurN gnome-themes-extras-0.9.0.orig/Nuvola/gtk-1.0/Makefile.am gnome-themes-extras-0.9.0/Nuvola/gtk-1.0/Makefile.am
--- gnome-themes-extras-0.9.0.orig/Nuvola/gtk-1.0/Makefile.am	2003-08-05 00:33:11.000000000 +0200
+++ gnome-themes-extras-0.9.0/Nuvola/gtk-1.0/Makefile.am	2005-12-15 19:18:52.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/Nuvola/gtk-1.0
+themedir = $(datadir)/themes/Nuvola/gtk
 theme_DATA = gtkrc
 
 EXTRA_DIST = $(theme_DATA)
diff -aurN gnome-themes-extras-0.9.0.orig/Nuvola/gtk-2.0/gtkrc gnome-themes-extras-0.9.0/Nuvola/gtk-2.0/gtkrc
--- gnome-themes-extras-0.9.0.orig/Nuvola/gtk-2.0/gtkrc	2005-03-23 16:43:21.000000000 +0100
+++ gnome-themes-extras-0.9.0/Nuvola/gtk-2.0/gtkrc	2005-12-15 19:19:56.000000000 +0100
@@ -6,7 +6,7 @@
 # A smooth blue theme modified from the Smooth-Winter theme, 
 # to match the Nuvola icon set
 
-gtk-icon-sizes = "panel-menu=32,32"
+gtk-icon-sizes = "panel-menu=24,24"
 
 style "default" {
 	######  GTK Style Flags  ###### 
