diff -Nuard gnome-themes-extras-0.3.orig/Gorilla/gtk-2.0/iconrc gnome-themes-extras-0.3/Gorilla/gtk-2.0/iconrc
--- gnome-themes-extras-0.3.orig/Gorilla/gtk-2.0/iconrc	2003-08-14 20:32:35.000000000 +0200
+++ gnome-themes-extras-0.3/Gorilla/gtk-2.0/iconrc	2003-09-18 17:44:48.000000000 +0200
@@ -171,5 +171,4 @@
 }
 
 
-
-class "GtkWidget" style "gorilla-default"
+class "GtkWidget" style "capplet-icons"
