diff -urN gnome-themes-extras-0.3.orig/Amaranth/icons/scalable/mimetypes/Makefile.in gnome-themes-extras-0.3/Amaranth/icons/scalable/mimetypes/Makefile.in
--- gnome-themes-extras-0.3.orig/Amaranth/icons/scalable/mimetypes/Makefile.in	Fri Aug 15 13:43:15 2003
+++ gnome-themes-extras-0.3/Amaranth/icons/scalable/mimetypes/Makefile.in	Fri Aug 15 13:45:08 2003
@@ -302,7 +302,7 @@
 
 install-data-local:
 	mkdir -p $(DESTDIR)$(icondir)
-	cd $(icondir) && \
+	cd $(DESTDIR)$(icondir) && \
 	ln -sf gnome-mime-audio.svg gnome-mime-application-x-ogg.svg
 
 uninstall-local:
