--- gnome-themes-extras-0.8.0/configure.in.orig	2004-08-31 17:20:55.000000000 +0200
+++ gnome-themes-extras-0.8.0/configure.in	2004-09-01 00:38:25.531203344 +0200
@@ -54,7 +54,7 @@
 AC_FUNC_ALLOCA
 AC_CHECK_FUNCS(setenv putenv)
 
-ALL_LINGUAS="az ca cs da de el en_CA en_GB es fr hr he it ja ko lt nl no pa pl pt pt_BR sk sq sr sr@Latn sv tr zh_CN zh_TW"
+ALL_LINGUAS="az ca cs da de el en_CA en_GB es fr hr he it ja ko lt nb nl pa pl pt pt_BR sk sq sr sr@Latn sv tr zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 AC_OUTPUT([
